.travis_script.sh

```diff
@@ -125,7 +125,6 @@ fi
 if [[ ${INTEGRATED} == 1 ]]
 then
     time make check-integrated-tests
-    time py.test-3 tools/pylib/
```

Comment:
Maybe keep this for zoidberg?
Or rebase #2214 to coord3d_merged2?

Reply:
I've dropped this because I'm assuming #2214 will go in at some point -- I'm not sure how best to handle the interaction with the 3D metrics branch. 

One way might be to merge #2214 now, make sure all the 3D metric stuff is in the zoidberg repo, and then merge next into 3D metrics?